From: Mike Kasick <mike@kasick.org>
Date: Wed, 19 Oct 2011 14:24:46 -0400

Set column_delay & timer_delay to xda recommended values.

Should replace with settings on custom ROMs when init.d is in place.

diff --git a/initramfs/init.rc b/initramfs/init.rc
--- a/initramfs/init.rc
+++ b/initramfs/init.rc
@@ -634,6 +634,9 @@
     chown system system /dev/dun
 # +++++++++++++++++++++++++++++++++++++++++++      
 
+    write /sys/devices/platform/s3c-keypad/column_delay 5
+    write /sys/devices/platform/s3c-keypad/timer_delay  5
+
     class_start default
 
 ## Daemon processes to be run by init.
